client/src/components/Chat/MessageCreator.js

```diff
@@ -0,0 +1,104 @@
+import React from 'react';
+import {
+    Box,
+    Button,
+    Container,
+    IconButton,
+    TextField
+} from '@material-ui/core';
+import { makeStyles } from '@material-ui/core/styles';
+import InsertDriveFileOutlinedIcon from '@material-ui/icons/InsertDriveFileOutlined';
+import SentimentSatisfiedOutlinedIcon from '@material-ui/icons/SentimentSatisfiedOutlined';
+
+const useStyles = makeStyles(theme => ({
+    container: {
+        minHeight: 120,
+        display: 'flex',
+        justifyContent: 'flex-start',
+        alignItems: 'center',
+        paddingLeft: theme.spacing(4),
+        '& p': {
+            fontSize: '1.375rem',
+        },
+    },
+    textContainer: {
+        width: '60%',
+        maxHeight: 100,
+        overflowY: 'auto',
+        '&::-webkit-scrollbar': {
+            width: '0.6rem',
+            height: '12rem',
+        },
+        '&::-webkit-scrollbar-track': {
+            boxShadow: 'inset 0 0 5px grey',
+            borderRadius: 4,
+        },
+        '&::-webkit-scrollbar-thumb': {
+            backgroundColor: 'rgba(0,0,0,.1)',
+            borderRadius: 4,
+        },
+        '&::-webkit-scrollbar-thumb:hover': {
+            background: '#ccc',
+        },
+    },
+    buttonContainer: {
+        width: '40%'
+    },
+    text: {
+        height: 100,
+        padding: 10
+    },
+    iconAdornments: {
+        fontSize: theme.spacing(3)
+    },
+    sendButton: {
+        margin: theme.spacing(3),
+        width: 110
+    },
```

Comment:
Here you can try:

    sendButton: {
        margin: theme.spacing(0, 3),
        width: 110,
    },

passing the height 0 spacing changes and only 3 to the width. It will clean up the oval hover.